configuration/pih/drugs/drugs.csv

```diff
@@ -150,121 +175,134 @@ af8c0f30-dfbe-11e9-8a34-2a2ae2dbcce4,20823,"Dextrose, 5% intravenous infusion, 1
-2ff6dfce-cd7e-4a1c-9916-f398c4aaec1f,10373,"Hydralazine hydrochloride, 25mg tablet",CIEL:77676,
-096dae85-9eee-466b-b059-9f5dd13af242,11190,"Hydralazine hydrochloride, Solution for injection, 20mg/mL, 1mL ampoule",CIEL:77676,
-ce857097-f7a1-4178-b018-a8067a5710d1,10579,"Hydrochlorothiazide (HCT), 25mg tablet",PIH:1243,
-a811008f-05fd-4f30-b50c-4d3636472db5,11632,"Hydrocortisone acetate, Skin cream, 1%, 15 gram tube",CIEL:77719,
+2ff6dfce-cd7e-4a1c-9916-f398c4aaec1f,11632,"Hydralazine hydrochloride, 25mg tablet",CIEL:77676,
+096dae85-9eee-466b-b059-9f5dd13af242,10724,"Hydralazine hydrochloride, Solution for injection, 20mg/mL, 1mL ampoule",CIEL:77676,
+ce857097-f7a1-4178-b018-a8067a5710d1,10739,"Hydrochlorothiazide (HCT), 25mg tablet",PIH:1243,
+a811008f-05fd-4f30-b50c-4d3636472db5,12367,"Hydrocortisone acetate, Skin cream, 1%, 15 gram tube",CIEL:77719,
 c6a8fca8-fce4-11e9-8f0b-362b9e155667,12390,"Hydrocortisone, Preservative free powder for solution for injection, 100mg vial",CIEL:77714,
 68f687c0-2875-4934-a060-825be42943e2,11136,"Hydrogen peroxide, 3%, 500mL bottle",PIH:313,
-0f0b85a4-b06b-484e-8213-3fe38279e3ad,10724,"Hydroxycarbamide, 500mg capsule",PIH:7658,
+0f0b85a4-b06b-484e-8213-3fe38279e3ad,10534,"Hydroxycarbamide, 500mg capsule",PIH:7658,
 74583b5e-80ec-4413-96b4-67907c98e443,10645,"Hyoscine butylbromide, 1.5mg (1mg/72hours) transdermal patch",PIH:431,
 8f2d1a7e-4ddb-4805-bf7e-1fb765592124,11380,"Hyoscine butylbromide, 10mg film coated tablet",PIH:431,
-9cfb1288-2db4-4ca6-b3f8-da4d85058e42,10739,"Hyoscine butylbromide, Solution for injection, 20mg/mL, 1mL ampoule",PIH:431,
+9cfb1288-2db4-4ca6-b3f8-da4d85058e42,11574,"Hyoscine butylbromide, Solution for injection, 20mg/mL, 1mL ampoule",PIH:431,
+6db68d7e-262a-4aaa-a669-f31b8758c5c1,14091,"Ibuprofen, 200mg tablet",PIH:912,
 36e09631-e6d8-41c9-92d2-c945750a39b9,10282,"Ibuprofen, 400mg tablet",PIH:912,
-49f646b5-c9a3-47e7-9fc2-bb187673b838,10534,"Ibuprofen, Oral solution, 20mg/mL, 100mL bottle",PIH:912,
+49f646b5-c9a3-47e7-9fc2-bb187673b838,10290,"Ibuprofen, Oral solution, 20mg/mL, 100mL bottle",PIH:912,
 78fb7988-dfbe-11e9-8a34-2a2ae2dbcce4,14149,"Ifosfamide, Powder for solution for injection, 2 gram vial",CIEL:77922,
 399834b0-b44b-11e3-a5e2-0800200c9a66,10967,"Imatinib, 400mg film coated tablet",PIH:7659,
 05868029-c96e-49c1-90b8-a35f8a4059c5,10762,"Imiquimod, Skin cream, 5%, 500mg sachet",CIEL:77956,
 af8c48f6-dfbe-11e9-8a34-2a2ae2dbcce4,10172,"Immunoglobulin (Human), Anti-D, 300 microgram (1,500 IU)/2mL prefilled syringe",CIEL:83322,
-d06a7e2c-948b-4b7e-86b6-52c36b536d49,10773,"Immunoglobulin (Human), Tetanus, 250 IU/mL, 1mL vial",CIEL:84878,
-af8c20ba-dfbe-11e9-8a34-2a2ae2dbcce4,11574,"Infant Formula, 0-6 months, Powder, 400 gram tin",CIEL:5254,
+d06a7e2c-948b-4b7e-86b6-52c36b536d49,10219,"Tetanus, Immunoglobulin, Human, 250IU/mL, 1mL vial",CIEL:84878,
+af8c20ba-dfbe-11e9-8a34-2a2ae2dbcce4,10442,"Infant Formula, 0-6 months, Powder, 400 gram tin",CIEL:5254,
```

Comment:
I think the Nav code should be 11574.  https://obnav.pih-emr.org/openboxes/inventoryItem/showStockCard/11574